src/main/java/seedu/foodrem/logic/parser/CliSyntax.java

```diff
@@ -11,12 +11,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ITEM_UNIT = new Prefix("u/");
     public static final Prefix PREFIX_ITEM_BOUGHT_DATE = new Prefix("bgt/");
     public static final Prefix PREFIX_ITEM_EXPIRY_DATE = new Prefix("exp/");
+    public static final Prefix PREFIX_ITEM_PRICE = new Prefix("p/");
     public static final Prefix PREFIX_ID = new Prefix("id/");
-
-    /* Definitions for Sort Command options */
-    public static final Prefix PREFIX_SORT_BY_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_SORT_BY_QTY = new Prefix("qty/");
-    public static final Prefix PREFIX_SORT_BY_BOUGHT_DATE = new Prefix("bgt/");
-    public static final Prefix PREFIX_SORT_BY_EXPIRY_DATE = new Prefix("exp/");
-    public static final Prefix PREFIX_SORT_BY_UNIT = new Prefix("unit/");
```

Comment:
I believe this is not needed. When we change a prefix in future, we would like to sort items by the new prefix as well.